src/Equality.js

```diff
@@ -0,0 +1,66 @@
+console.log("Equality (==)");
+
+console.log(1 == 1);
+// expected output: true
+
+console.log('hello' == 'hello');
+// expected output: true
+
+console.log('1' ==  1);
+// expected output: true
+
+console.log(0 == false);
+// expected output: true
+
+
+console.log("Strict equality (===)");
+
+console.log(1 === 1);
+// expected output: true
+
+console.log('hello' === 'hello');
+// expected output: true
+
+console.log('1' ===  1);
+// expected output: false
+
+console.log(0 === false);
```

Comment:
Might be nice for each of these examples to above console.log the calculation you're doing as a string (e.g. `console.log("0 == false"`) so when someone runs it they can see:
> 0 == "false"
> true

Reply:
That is a nice ideas I will do it